src/GovUk.Education.ExploreEducationStatistics.Content.Model/Repository/ReleaseRepository.cs

```diff
@@ -53,148 +53,151 @@ await _contentDbContext.Entry(release)
 
     public async Task<Release?> GetLatestPublishedReleaseVersion(Guid publicationId)
     {
-        var publication = await _contentDbContext.Publications
-            .Include(p => p.LatestPublishedRelease)
-            .SingleOrDefaultAsync(p => p.Id == publicationId);
-        return publication?.LatestPublishedRelease;
+        return (await GetLatestReleaseVersionsQueryable(publicationId, publishedOnly: true)
+                .ToListAsync())
+            .OrderByReverseChronologicalOrder()
+            .FirstOrDefault();
     }
 
     public async Task<Release?> GetLatestReleaseVersion(Guid publicationId)
     {
-        var releases = await ListLatestReleaseVersions(publicationId);
-        return releases.FirstOrDefault();
+        return (await GetLatestReleaseVersionsQueryable(publicationId)
+                .ToListAsync())
+            .OrderByReverseChronologicalOrder()
+            .FirstOrDefault();
```

Comment:
Discussed with Ben on a call

Reply:
Had a call with @mmyoungman to talk about the reason for this. It's because we can't order releases by time in SQL as the time period order is an Enum rather than a reference table in the database and I didn't want to change that as part of this work.